### DISCUSSION TITLE
Ram Timings for Version 1.9.0+

### DISCUSSION THREAD

**Discussion by hanai3Bi** [hanai3Bi/Switch-OC-Suite#54] [Ram Timing | Customization]
These are most recent timing customizations i use
Samsung MGCJ: 1200/650mV, 4-3-4-5-3-5-3
Samsung MGCL: 1175/600mV, 4-3-4-5-4-5-4

**Comment by jang2ah2** [comment]
Erista?

**Reply by hanai3Bi** [reply]
i don't actualy use erista, so i don't know timings for erista.
see timings submitted by others, or come to our discord and ask other erista guys
